pkg/controllers/clusterresourceplacement/clusterresourceplacement_controller.go

```diff
@@ -21,6 +22,58 @@ func (r *Reconciler) Reconcile(_ context.Context, _ controller.QueueKey) (ctrl.R
 	return ctrl.Result{}, nil
 }
 
+func (r *Reconciler) handleDelete(ctx context.Context, crp *fleetv1beta1.ClusterResourcePlacement) (ctrl.Result, error) {
+	crpKObj := klog.KObj(crp)
+	if !controllerutil.ContainsFinalizer(crp, fleetv1beta1.ClusterResourcePlacementCleanupFinalizer) {
```

Comment:
Shouldn't the DeletionTimestamp be checked here? Not having a finalizer != the CRP is being deleted I think.

Reply:
Or is it being done at the upper layer? The Reconcile function isn't updated so I am a little bit confused.

Reply:
DeletionTimestamp will be checked in the caller. updated the log.